.github/workflows/android-cd.yml

```
@@ -0,0 +1,107 @@
+            -   name: Upload Release Build to Artifacts
+                uses: actions/upload-artifact@v4
+                with:
+                    name: release-artifacts
+                    path: app/build/outputs/bundle/release/
+                    if-no-files-found: error
+
+            -   name: Create Github Release
+                uses: softprops/action-gh-release@v2
+                with:
+                    tag_name: ${{ steps.extract_version.outputs.version }}
+                    release_name: ${{ steps.extract_version.outputs.version }}
```

actionlint (1.7.7)

input "release_name" is not defined in action "softprops/action-gh-release@v2". available inputs are "append_body", "body", "body_path", "discussion_category_name", "draft", "fail_on_unmatched_files", "files", "generate_release_notes", "make_latest", "name", "prerelease", "preserve_order", "repository", "tag_name", "target_commitish", "token"

(action)